content/en/opentelemetry/setup/ddot_collector/install/kubernetes_gateway.md

```diff
@@ -539,11 +546,19 @@ For advanced scenarios, you can deploy multiple gateway layers to create a proce
                 exporters: [otlp]
     ```
     
+## View gateway pods on Fleet Automation
+
+By default, the gateway DDOT collectors include a [Datadog extension][11]. The Datadog Extension enables collector configuration and build information to be viewed in both Datadog Infrastructure Monitoring and Fleet Automation. The gateway pods can be viewed on the Integrations -> Fleet Automation page.
+
+{{< img src="opentelemetry/embedded_collector/fleet_automation2.png" alt="Fleet Automation page showing DDOT gateway pods" style="width:100%;" >}}
+
+You can click on any gateway pod and see the detailed build information and collector configurations.
+
+{{< img src="opentelemetry/embedded_collector/fleet_automation3.png" alt="Fleet Automation page showing the collector config of one DDOT gateway pod" style="width:100%;" >}}
+
 ## Known limitations
 
-  * **Gateway pods on Fleet Automation**: Standalone gateway pods are not yet visible on the Fleet Automation page. Only DaemonSet Collectors are displayed. This is being actively addressed.
```

Comment:
This bug and the one on line 546 are fixed in the latest version so I removed them. Let me know if it's better to keep them with a note that they're fixed in 3.156.0 and above.